app/models/concerns/subject_counts.rb

```diff
@@ -10,7 +10,7 @@ def retired_subjects_count
   end
 
   def finished?
-    @finished ||= if subject_sets.empty?
+    @finished ||= if subject_sets.empty? || subjects_count == 0
```

Comment:
this is to cover the case when 0 == 0 and when owners delete subjects after a data run.